test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -199,3 +255,38 @@ func runCpaSampleCode(ctx context.Context, f *framework.Framework, runTests int,
 	err = e2epod.WaitForPodSuccessInNamespaceTimeout(ctx, f.ClientSet, pod.ObjectMeta.Name, f.Namespace.Name, 300*time.Second)
 	gomega.Expect(err).To(gomega.BeNil(), utils.GetPodLogs(ctx, f, pod.ObjectMeta.Name, pod.Spec.Containers[0].Name))
 }
+
+func injectError(ctx context.Context, f *framework.Framework, resourceName v1.ResourceName) {
+	nodeName, _ := utils.FindNodeAndResourceCapacity(f, ctx, resourceName.String())
```

Comment:
I think nodeName should be checked for emptiness.

Reply:
Done!